bitnami/kafka/templates/servicemonitor-jmx-metrics.yaml

```
@@ -30,6 +30,14 @@ spec:
       {{- if .Values.metrics.serviceMonitor.scrapeTimeout }}
       scrapeTimeout: {{ .Values.metrics.serviceMonitor.scrapeTimeout }}
       {{- end }}
+      {{- if .Values.metrics.serviceMonitor.relabelings }}
+      relabelings:
+      {{- toYaml .Values.metrics.serviceMonitor.relabelings | nindent 6 }}
```

In this case, it is best if we include it in only one line. That's how we are doing it across other charts (and `nindent` will insert the newline):

```suggestion
      relabelings: {{- toYaml .Values.metrics.serviceMonitor.relabelings | nindent 6 }}
```

---

Fix!